documentation/modules/ref-jvm-options.adoc

```diff
@@ -122,3 +122,24 @@ jvmOptions:
   gcLoggingEnabled: true
 # ...
 ----
+
+=== Output Garbage collector logs to a file
+
+[CAUTION]
+====
+CAUTION HERE
```

Comment:
I think we need to write the actual warning here ... about the user needing to maintain the file etc.

Reply:
Apologies, this was more of a self “todo” and wouldn’t be in the final version of this PR

Reply:
No worries - feel free to change it once the discussion with @ppatierno and @tombentley clarifies. I was just going through the github notifications so I was looking through this PR again.